reference/3.0/Microsoft.PowerShell.Utility/Invoke-WebRequest.md

```diff
@@ -36,110 +36,130 @@ This cmdlet was introduced in Windows PowerShell 3.0.
 > script code in the web page may be run when the page is being parsed to populate the `ParsedHtml` property.
 > Use the `-UseBasicParsing` switch to suppress this.
 
-
 ## EXAMPLES
 
-### Example 1
+### Example 1: Send a web request
```

Comment:
This example doesn't really work because the format of the result pages from Bing and Google have changed. We need a better, simple example.